business/mesh.go

```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/kiali/kiali/util/httputil"
 )
 
+const DefaultClusterID = "Kubernetes"
```

Comment:
are cluster IDs typically mixed case? I would think they would be all lowercase. If so, I would change this.

Not sure if it matters but... it would be consistent with "real" cluster IDs anyway.

```suggestion
const DefaultClusterID = "kubernetes"
```

Reply:
No, do not make this change.  "Kubernetes" is correct, we are mirroring the value set by Istio as the default.